multi-arch-builders/README.md

```diff
@@ -127,22 +127,6 @@ ibmcloud is instance-create $NAME $VPC $ZONE $PROFILE $SUBNET --output json --im
      --user-data @fcos-s390x-builder.ign > out.json
```

Comment:
The argument to ssh keys '--keys $SSHKEY' (can be obtained using 'ibmcloud is keys') is missing. is this expected?

Reply:
In this case we are passing SSH keys via Ignition, so no need to use the cloud keys. See 

https://github.com/coreos/fedora-coreos-pipeline/blob/f3f6c5ebf7688b1f91ff3e29e1d215bceea2d727/multi-arch-builders/fcos-s390x-builder.bu#L19-L26

We can update that list as needed.

Reply:
thank you for pointing me to this.